sdk/objc/components/audio/RTCAudioSession.mm

```diff
@@ -494,9 +496,16 @@ - (void)handleRouteChangeNotification:(NSNotification *)notification {
     case AVAudioSessionRouteChangeReasonCategoryChange:
```

Comment:
FYI, these events are only triggered after the AVAudioSession has been activated. Initially it remains inactive until WebRTC decides to start it.